tests/lax_numpy_test.py

```diff
@@ -2040,6 +2040,8 @@ def testRollaxis(self, shape, dtype, start, axis, rng_factory):
       for axis in [None, 0, 1, -2, -1]
       for rng_factory in [jtu.rand_some_zero]))
   def testPackbits(self, shape, dtype, axis, bitorder, rng_factory):
+    if numpy_version < (1, 17, 0):
```

Comment:
Shouldn't we be testing the other arguments (i.e., the `bitorder=None` case?)

Reply:
Turns out `bitorder` should only be "big" or "little" and defaults to "big".